example-consuming-client/src/api.ts

```diff
@@ -1,13 +1,18 @@
-import GraphQLRestRouter, { OpenApi } from 'graphql-rest-router';
+import GraphQLRestRouter, { OpenApi, InMemoryCache } from 'graphql-rest-router';
 import SCHEMA from './schema';
 
 const { ENDPOINT = '' } = process.env;
 
-const api = new GraphQLRestRouter(ENDPOINT, SCHEMA);
+const api = new GraphQLRestRouter(ENDPOINT, SCHEMA, {
+  cacheEngine: new InMemoryCache(),
+
+  defaultCacheTimeInMs: 3000,
```

Comment:
Ensure this is hooked up to `cacheEngine` (currently done per route but should be universal?)